rcnn/rcnn.cpp

```diff
@@ -18,10 +18,16 @@ static const std::vector<float> PIXEL_STD = {1.0, 1.0, 1.0};
 static constexpr float MIN_SIZE = 800.0;
 static constexpr float MAX_SIZE = 1333.0;
 static constexpr int NUM_CLASSES = 80;
-static constexpr int INPUT_H = 480;
-static constexpr int INPUT_W = 640;
-static int IMAGE_HEIGHT = 800;
-static int IMAGE_WIDTH = 1333;
+static int INPUT_H;  // size of model input
+static int INPUT_W;
+static constexpr int INPUT_H_ = 480;  // size of original image
+static constexpr int INPUT_W_ = 640;
+static int IMAGE_HEIGHT;  // size of model input
```

Comment:
looks like IMAGE_HEIGHT are same with INPUT_H, one left will be better if so

Reply:
Yes, these two vars are same. I'll leave only one var.